crates/matrix-sdk-crypto/src/identities/manager.rs

```diff
@@ -1210,4 +1211,23 @@ pub(crate) mod tests {
         assert_eq!(devices.devices().count(), 1);
         assert_eq!(devices.devices().next().unwrap().device_id(), "LVWOVGOXME");
     }
+
+    #[async_test]
+    async fn identities_stream() {
+        let manager = manager().await;
+        let (request_id, _) = manager.build_key_query_for_users(vec![user_id()]);
+
+        let stream = manager.store.user_identities_stream();
+        pin_mut!(stream);
+
+        manager.receive_keys_query_response(&request_id, &own_key_query()).await.unwrap();
+
+        let update = stream
+            .next()
+            .now_or_never()
+            .unwrap()
+            .expect("The broadcaster should have sent out a identities update");
```

Comment:
I added `assert_ready!` to `stream-assert` for this a while ago, but just realized I hadn't released it – fixed that now.